docs/docs/guides/quick-start/gather-insights.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Gather insights
+description: Learn how to gather insights about your project.
+---
+
+# Gather insights
+
+Tuist can integrate with a server to extend its capabilities. One of those capabilities is gathering insights about your project and builds. All you need is to have an account with a project in the server. 
+
+First of all, you'll need to authenticate by running:
+
+```bash
+tuist auth
+```
+
+## Create a project
+
+You can then create a project by running:
+
+```bash
+tuist project create MyApp -o my-handle
+
+# Tuist project tuist/myapp was successfully created 🎉
+```
+
+Copy `tuist/myapp`, which represents the full handle of the project.
+
+## Connect projects
+
+After creating the project on the server, you'll have to connect it to your local project. Run `tuist edit` and edit the `Config.swift` file to include the full handle of the project:
+
+```swift
+import ProjectDescription
+
+let config = Config(fullHandle: "tuist/myapp")
+```
+
+Voilà! You're now ready to gather insights about your project and builds. Run `tuist test` to run the tests reporting the results to the server.
```

Comment:
It's probably worth mentioning that locally, the reports might not get immediately published (whereas they will get immediately published on the CI).